docs/source/documentation/components/sample-code/_images.html

```diff
@@ -1,17 +1,17 @@
-<img srcset="../assets/img/image-1920-xlg.jpg 192w,
-             ../assets/img/image-960-lg.jpg   960w,
-             ../assets/img/image-480-md.jpg   480w"
-     src="../assets/img/image.jpg"
+<img srcset="{{relativePath}}assets/img/image-1920-xlg.jpg 192w,
+             {{relativePath}}assets/img/image-960-lg.jpg   960w,
+             {{relativePath}}assets/img/image-480-md.jpg   480w"
+     src="{{relativePath}}assets/img/image.jpg"
      alt="24 column or fulscreen image.">
 
-<img srcset="../assets/img/image-960-lg.jpg   960w,
-             ../assets/img/image-480-md.jpg   480w,
-             ../assets/img/image-240-sm.jpg   240w"
-     src="../assets/img/image.jpg"
+<img srcset="{{relativePath}}assets/img/image-960-lg.jpg   960w,
+             {{relativePath}}assets/img/image-480-md.jpg   480w,
+             {{relativePath}}assets/img/image-240-sm.jpg   240w"
+     src="{{relativePath}}assets/img/image.jpg"
      alt="12 column or two up image.">
 
-<img srcset="../assets/img/image-640-bmd.jpg  640w,
-             ../assets/img/image-320-smd.jpg  320w,
-             ../assets/img/image-160-xsm.jpg  160w"
-     src="../assets/img/image.jpg"
+<img srcset="{{relativePath}}assets/img/image-640-bmd.jpg  640w,
```

Comment:
this still won't work, these images need to be moved to a folder that is version controlled, yes?